src/overlays/actors/ovl_Bg_Kin2_Bombwall/z_bg_kin2_bombwall.c

```diff
@@ -28,50 +40,181 @@ const ActorInit Bg_Kin2_Bombwall_InitVars = {
     (ActorFunc)BgKin2Bombwall_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80B6E6F0 = {
+ColliderCylinderInit D_80B6E6F0 = {
```

Comment:
Ah you see, at the top here this commented out line 
```
 // static ColliderCylinderInit sCylinderInit = {
```
That is a suggested name for this data once you got the file to match, they gave you the version with default data values to start with `D_80B6E6F0` but now we can name it to something for clarity.